README.md

```diff
@@ -57,26 +57,16 @@ To get the source code from GitHub, type
     cd ./azure-sdk-for-php
 
 > **Note**
-> 
+>
 > The PHP Client Libraries for Microsoft Azure have a dependency on the [HTTP_Request2](http://pear.php.net/package/HTTP_Request2), [Mail_mime](http://pear.php.net/package/Mail_mime), and [Mail_mimeDecode](http://pear.php.net/package/Mail_mimeDecode) PEAR packages. The recommended way to resolve these dependencies is to install them using the [Composer package manager](http://getcomposer.org).
 
-
 ##Install via Composer
+1. Make sure you have installed **[Compoeser](https://getcomposer.org/doc/00-intro.md#introduction)** in your system.
+It can be installed locally or globally depending on your own needs.
 
-* Create a file named **composer.json** in the root of your project and add the following code to it:
-```json
-    {
-        "require": {        
-            "microsoft/windowsazure": "^0.4"
-        }  
-    }
-```
-
-* Download **[composer.phar](http://getcomposer.org/composer.phar)** in your project root.
-
-* Open a command prompt and execute this in your project root
+2. Execute the following command in order to get this package:
 
-    php composer.phar install
+    php composer create-project microsoft/windowsazure /path/to/install/dir ^0.4
```

Comment:
@yaqiyang, I'll try to address your comments one by one:

1. as I have composer installed globally and my executable is called `composer`, I don't need to suffix the call with `.phar`:
```
$ which composer
/usr/local/bin/composer
```
Obviously, it depends on which mechanism the user chooses based on the install documentation: https://getcomposer.org/doc/00-intro.md

2. I have no issues installing `0.4.2` version with this command:
```
$ composer create-project microsoft/windowsazure testing-resolved-version ^0.4
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
Installing microsoft/windowsazure (v0.4.2)
  - Installing microsoft/windowsazure (v0.4.2)
    Loading from cache

Created project in testing-resolved-version
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
  - Installing firebase/php-jwt (v3.0.0)
    Loading from cache

  - Installing pear/pear_exception (v1.0.0)
    Loading from cache

  - Installing pear/net_url2 (v2.2.0)
    Loading from cache

  - Installing pear/http_request2 (v2.3.0)
    Loading from cache

  - Installing pear/console_getopt (v1.4.1)
    Loading from cache

  - Installing pear/pear-core-minimal (v1.10.1)
    Loading from cache

  - Installing pear/mail_mime (1.10.0)
    Loading from cache

  - Installing pear/mail_mime-decode (1.5.5.2)
    Loading from cache

  - Installing mikey179/vfsstream (v1.6.2)
    Loading from cache

  - Installing phpdocumentor/reflection-docblock (2.0.4)
    Downloading: 100%         

  - Installing phpunit/php-token-stream (1.4.8)
    Downloading: 100%         

  - Installing symfony/yaml (v3.0.4)
    Loading from cache

  - Installing sebastian/version (1.0.6)
    Loading from cache

  - Installing sebastian/global-state (1.1.1)
    Downloading: 100%         

  - Installing sebastian/recursion-context (1.0.2)
    Downloading: 100%         

  - Installing sebastian/exporter (1.2.1)
    Downloading: 100%         

  - Installing sebastian/environment (1.3.5)
    Loading from cache

  - Installing sebastian/diff (1.4.1)
    Downloading: 100%         

  - Installing sebastian/comparator (1.2.0)
    Downloading: 100%         

  - Installing phpunit/php-text-template (1.2.1)
    Downloading: 100%         

  - Installing doctrine/instantiator (1.0.5)
    Downloading: 100%         

  - Installing phpunit/phpunit-mock-objects (2.3.8)
    Loading from cache

  - Installing phpunit/php-timer (1.0.7)
    Downloading: 100%         

  - Installing phpunit/php-file-iterator (1.4.1)
    Downloading: 100%         

  - Installing phpunit/php-code-coverage (2.2.4)
    Loading from cache

  - Installing phpspec/prophecy (v1.6.0)
    Loading from cache

  - Installing phpunit/phpunit (4.8.24)
    Loading from cache

phpdocumentor/reflection-docblock suggests installing dflydev/markdown (~1.0)
phpdocumentor/reflection-docblock suggests installing erusev/parsedown (~1.0)
sebastian/global-state suggests installing ext-uopz (*)
phpunit/phpunit suggests installing phpunit/php-invoker (~1.1)
Generating autoload files
```

3. I don't understand your point, what you mean with "so many hoops"? I don't share your conception about the README file, since I think it isn't only for beginners but a documentation piece that must cover the main concepts of the project which is described within it.